docker-compose.fvm.yml

```diff
@@ -23,6 +23,23 @@ services:
     depends_on:
       - db
 
+  storage:
+    image: node:18-alpine3.14
+    restart: always
+    command: sh -c "npm i && npm run  build && npm run start"
+    stdin_open: true
+    tty: true
+    ports:
+      - 5555:5555
```

Comment:
For security reason here, we should not enable public access.

Reply:
ok kub